Kitodo/src/main/java/org/kitodo/production/interfaces/activemq/ActiveMQDirector.java

```diff
@@ -23,10 +23,6 @@
 import javax.jms.MessageConsumer;
 import javax.jms.MessageProducer;
 import javax.jms.Session;
-import javax.servlet.ServletContextEvent;
-import javax.servlet.ServletContextListener;
-import javax.servlet.annotation.WebListener;
-
```

Comment:
This empty line should probably be re-instated to fulfill Checkstyle requirements.